### PR TITLE
Fix: Remove fast element composedParent & composedContains exports from foundation

### PR DESCRIPTION
**Pull request by tebinraouf**
# Pull Request

## 📖 Description

While using the Foundation components in Angular, the angular project throws an error as noted https://github.com/microsoft/fast/issues/6070#issuecomment-1271066595. Since `composedParent` and `composedContains` are not needed by the CLI, this PR removes them.

### 🎫 Issues

Fixes https://github.com/microsoft/fast/issues/6070#issuecomment-1271066595

## 📑 Test Plan

All current tests from Foundation are passing as expected. No new tests are added.

### General

- [x] I have included a change request file using `$ yarn change`
- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](/docs/community/code-of-conduct/#our-standards) for this project.

### CODE REVIEW

**Comment by tebinraouf** [comment]
> @microsoft-github-policy-service agree

@microsoft-github-policy-service agree

**Comment by tebinraouf** [comment]
@microsoft-github-policy-service agree

File: change/@microsoft-fast-foundation-af86d18f-3e01-482d-b150-9b167c1c1828.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "remove unnecessary composedParent & composedContains exports from Foundation",
+  "packageName": "@microsoft/fast-foundation",
+  "email": "tebin.raouf@thomsonreuters.com",
+  "dependentChangeType": "patch"
```
Comment (nicholasrice) [inline change/@microsoft-fast-foundation-af86d18f-3e01-482d-b150-9b167c1c1828.json:6]:
```suggestion
  "dependentChangeType": "prerelease"
```

**Review by nicholasrice — APPROVED** [review]
Looks good, just one issue noted. Thanks!

**Review by EisenbergEffect — APPROVED** [review]

**Comment by tebinraouf** [comment]
Please let me know if I need to address the failing tests or this will be merged regardless.

**Comment by EisenbergEffect** [comment]
@tebinraouf I'm looking into it. Will let you know.

**Comment by EisenbergEffect** [comment]
Thanks @tebinraouf !

**Pull request merged**